src/en/readcomic/src/eu/kanade/tachiyomi/extension/en/readcomic/ReadComic.kt

```diff
@@ -0,0 +1,226 @@
+                    is GenreFilter -> addQueryParameter("wg", filter.toUriPart())
+                    else -> {}
+                }
+            }
+            if (page > 1) addQueryParameter("page", page.toString())
+        }.build()
+        return GET(url, headers)
+    }
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        setUrlWithoutDomain(element.select("div.egb-right > a.egb-serie").attr("href"))
+        title = element.select("div.egb-right > a.egb-serie").text()
+        thumbnail_url = element.select("a.eg-image > img").attr("src")
+    }
+
+    override fun latestUpdatesFromElement(element: Element) = SManga.create().apply {
+        val selector = "ul.line-list > li > a.big-link"
+        setUrlWithoutDomain(element.select(selector).attr("href"))
+        title = element.select(selector).text()
+        thumbnail_url = fetchThumbnailURL(element.select(selector).attr("href"))
+    }
+    override fun searchMangaFromElement(element: Element) = SManga.create().apply {
+        setUrlWithoutDomain(element.select("div.dlb-right > a.dlb-title").attr("href"))
+        title = element.select("div.dlb-right > a.dlb-title").text()
+        thumbnail_url = element.select("a.dlb-image > img").attr("src")
+    }
+    private fun fetchThumbnailURL(url: String) = client.newCall(GET(url, headers)).execute().asJsoup().select("div.anime-image > img").attr("src")
+
+    // private fun fetchPagesFromNav(url: String) = client.newCall(GET(url, headers)).execute().asJsoup()
+
+    override fun popularMangaNextPageSelector() = "div.general-nav > a:contains(Next)"
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun searchMangaSelector() = "div.dl-box"
+
+    override fun searchMangaNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun mangaDetailsParse(document: Document): SManga {
+        return SManga.create().apply {
+            title = document.select("h1.title").text()
+            thumbnail_url = document.select("div.anime-image > img").attr("src")
+            status = parseStatus(document.select("ul.anime-genres li.status").text())
+            author = document.select("td:contains(Author:) + td").text()
+            description = document.select(".detail-desc-content > p").text()
+            genre = document.select("ul.anime-genres > li > a[href*='genre']").joinToString { it.text() }
+        }
+    }
+
+    private fun parseStatus(element: String): Int = when {
+        element.contains("Completed") -> SManga.COMPLETED
+        element.contains("Ongoing") -> SManga.ONGOING
+        else -> SManga.UNKNOWN
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val document = response.asJsoup()
+        val chapters = ArrayList<SChapter>()
+
+        document.select(chapterListSelector()).forEach {
+            chapters.add(chapterFromElement(it))
+        }
+
+        return chapters
+    }
```

Comment:
you are not doing anything special here, so the default implementation should suffice. remove it